[
  {
    "poster": {
      "poster_name": "eljie",
      "poster_handle": "eljie"
    },
    "time_stamp": "2024-09-12 08:08:11",
    "text": ":wave: Hello, team!"
  },
  {
    "poster": {
      "poster_name": "eljie",
      "poster_handle": "eljie"
    },
    "time_stamp": "2024-09-12 08:09:53",
    "text": "I would like to ask if what possible issue im encountering atm when having this error\n```Failed to refresh HubSpot credentials: HTTPSConnectionPool(host='<http://api.hubapi.com|api.hubapi.com>', port=443): Max retries exceeded with url: /oauth/v1/token ```\nThe app is working fine with my local but having issues on aws lambda"
  },
  {
    "poster": {
      "poster_name": "eljie",
      "poster_handle": "eljie"
    },
    "time_stamp": "2024-09-12 08:11:23",
    "edited": {
      "time_stamp": "2024-09-12 08:11:46"
    },
    "text": "Note that my app's region is on `eu-west-2`  London at aws"
  }
]